contrib/CMakeLists.txt

```diff
@@ -102,3 +102,9 @@ if(UNIX)
     DESTINATION ${CUBRID_DATADIR}/install/)
 endif(UNIX)
 
+# multi-process unloaddb scripts
+if(UNIX)
+  install(PROGRAMS
+    ${CMAKE_SOURCE_DIR}/contrib/script/unloaddb.sh.sh
```

Comment:
Invalid path. (script/unloaddb.sh.sh -> scripts/unloaddb.sh)

Reply:
Fixed. Thank you!